LayoutTests/platform/ios/TestExpectations

```diff
@@ -3659,3 +3659,7 @@ webkit.org/b/241048 imported/w3c/web-platform-tests/html/canvas/element/manual/w
 webkit.org/b/241205 fast/forms/textfield-outline.html [ Pass Failure ]
 
 fast/text/bulgarian-system-language-shaping.html [ Pass ]
+
+fast/text/accessibility-bold-system-font.html [ Pass ]
```

Comment:
Might be a little nicer to move these tests into their own dir so you can just pass the whole dir.